src/olympia/legacy_discovery/modules.py

```diff
@@ -66,15 +65,23 @@ class MonthlyPick(TemplatePromo):
     slug = 'Monthly Pick'
     template = 'legacy_discovery/modules/monthly.html'
 
+    def get_pick(self, locale):
+        monthly_pick = MP.objects.filter(locale=locale)[0]
```

Comment:
"`MP`" for a variable name 🙄

Reply:
I hate it too. The rationale is that the model name is `MonthlyPick` but it's also the name of the promo module. Changing promo modules names is tricky (the enabled modules names are stored in the database) so I didn't touch it...

I guess I could rename it to `MonthlyPickModel` instead. Quite a mouthful but descriptive...

Reply:
or `from olympia.bandwagon import models`, I guess.  But your rename works too.